keras2onnx/_parse_tf.py

```diff
@@ -210,8 +210,6 @@ def build_layer_outputs(model, graph, outputs):
                     # fx_[1] is output node redirect function.
                     output_node = fx_list[1](lobj, op_)
                     output_node = _advance_output_node_if_successor(graph, ln_, output_node)
-                    assert graph.get_operation_by_name(output_node) is not None, "Parsing layer({}) failed.".format(
```

Comment:
This is an important check to make sure the model be converted correctly. need to debug why it is violated.

Reply:
Find that the layer spec converter for DepthwiseConv2D, there is no output called 'BiasAdd', but only 'depthwise'. So I will change this.